include/c2pa.h

```diff
@@ -84,6 +152,16 @@ IMPORT extern char *c2pa_version(void);
  */
 IMPORT extern char *c2pa_error(void);
 
+/**
+ * Load Settings from a string
+ * # Errors
+ * Returns -1 if there were errors, otherwise returns 0
+ * The error string can be retrieved by calling c2pa_error
+ * # Safety
+ * Reads from null terminated C strings
```

Comment:
Grammar nit (sorry, yes, my mom was an English teacher): When two words come together to form a single adjective, they are typically hyphenated (as in "null-terminated"). Applies throughout the PR.

https://apastyle.apa.org/style-grammar-guidelines/spelling-hyphenation/hyphenation

Reply:
So should this be NULL-terminated or null-terminated?